test/e2e/utils.go

```diff
@@ -33,6 +34,9 @@ func writePodsLogs(namespace string, writer io.Writer) error {
 	podsClientset := framework.Global.KubeClient.CoreV1().Pods(namespace)
 
 	for _, pod := range podList.Items {
+		if !strings.Contains(pod.Name, "nmstate") {
```

Comment:
doesn't seem stable. can we check against some labels instead? maybe we have "app" label set to nmstate

Reply:
Good idea, Done.